Makefile

```diff
@@ -1,10 +1,19 @@
 JUNIT_OUTPUT_DIR = $(dir $(JUNIT_OUTPUT_FILE))
 
-test: build
-	docker run --rm cors-proxy-app /tmp/scripts/run --daemon
+BUILDER_IMAGE ?= quay.io/3scale/s2i-openresty-centos7:1.13.6.1-rover5
+RUNTIME_IMAGE ?= $(BUILDER_IMAGE)-runtime
+IMAGE_NAME ?= cors-proxy-candidate
 
-build: rock
-	s2i build . quay.io/3scale/s2i-openresty-centos7:1.11.2.5-1-rover2 cors-proxy-app
+BUILD_TYPE ?= builder
+
+test: build-$(BUILD_TYPE)
+	docker run --rm $(IMAGE_NAME) sh -c 'exec $$([[ -f /tmp/scripts/run ]] && echo /tmp/scripts/run || echo /opt/app-root/scripts/run) --daemon'
```

Comment:
Would be nice to unify this path in the s2i-openresty image. Not sure how easy it is because that is done by s2i. Contributions welcome.

Reply:
I will try to take a look at it :)